### DISCUSSION TITLE
The documentation lacks a description of support for the Phase 2 proposal

### DISCUSSION THREAD

**Discussion by Iceber** [WasmEdge/WasmEdge#2115] [Show and tell]
https://github.com/WasmEdge/WasmEdge/blob/af096e769bb1afcc8aa5a848d6ab33bdbb8c2ea1/docs/book/en/src/features/proposals.md?plain=1#L44-L56

The documentation only describes which Feature proposals are supported, does it need to be explicitly stated that wasmedge supports all [Phase 2 proposals](https://github.com/WebAssembly/WASI/blob/main/Proposals.md#phase-2---proposed-spec-text-available-cg--wg)?

**Comment by hydai** [comment]
Hi @Iceber, 
You can assume that WasmEdge doesn't support any proposals not listed on that list. I would like to know the motivation for this.

@q82419 
Should we add all proposals listed on the WASM/WASI proposal pages?
It looks like we've missed some of them on the list.
